app/controllers/concerns/decidim/consultations/needs_question.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+module Decidim
```

Comment:
This concern already exists in decidim decidim, You can make a decorator by adding only the necessary functionality

Reply:
As commented face to face, we will overwrite this concern for now.